tests/llms/test_litellm.py

```diff
@@ -152,3 +152,52 @@ def test_chat_model_basic(monkeypatch: MonkeyPatch) -> None:
 def test_metadata() -> None:
     llm = LiteLLM(model="gpt-3.5-turbo")
     assert isinstance(llm.metadata.context_window, int)
+
+# Extra local LiteLLM Tests
```

Comment:
tbh I would prefer these to be proper tests that are skipped when things aren't installed?